packages/list_bloc/lib/src/cubits/list_cubit.dart

```diff
@@ -21,4 +21,50 @@ class ListCubit<T, F> extends DataCubit<List<T>, F> {
     }
   }
 // TODO(alexis): implement remove, add, replace etc
+
+  Future<void> remove(T item) async {
+    try {
+      final data = <T>[...state.data ?? []];
+      if (data.isNotEmpty) {
+        data.remove(item);
+        emit(
```

Comment:
What if after removing the item it becomes empty?

Reply:
I have updated it you can review it again